build.sh

```diff
@@ -17,11 +17,7 @@ if [ "$1" = "windows" ]; then
 
 elif [ "$1" = "linux" ]; then
   set -x
-  docker build -t oonibuild .
-  docker run -v `pwd`:/oonibuild -w /oonibuild -t --cap-drop=all               \
-    --user `id -u`:`id -g` -e 'GOCACHE=/tmp/go/cache' -e 'GOPATH=/tmp/go/path' \
-    oonibuild                                                                  \
-    go build $buildtags -ldflags="$ldflags"                                    \
+  $0 __docker go build $buildtags -ldflags="$ldflags"                          \
```

Comment:
Code to build docker container and to run the container has been factored into a sub command.